src/net_processing.cpp

```diff
@@ -1925,64 +1936,67 @@ static void ProcessHeadersMessage(CNode& pfrom, CConnman* connman, ChainstateMan
     return;
 }
 
-void static ProcessOrphanTx(CConnman* connman, CTxMemPool& mempool, std::set<uint256>& orphan_work_set, std::list<CTransactionRef>& removed_txn) EXCLUSIVE_LOCKS_REQUIRED(cs_main, g_cs_orphans)
+/**
+ * Reconsider orphan transactions after a parent has been accepted to the mempool.
+ *
+ * @param[in]      connman          Reference to the connection manager
+ * @param[in]      mempool          Reference to mempool
```

Comment:
2ec4eead [in/out]? (`mempool` needs to be mutable)

Reply:
hmmm I'm not sure. mempool isn't used by the caller after the function returns, so I'm not sure I'd call it an 'out param'. Just because an argument is non-const, I don't think that automatically makes it an out-param.